rotkehlchen/db/schema.py

```diff
@@ -817,6 +817,34 @@
 );
 """  # noqa: E501
 
+# Lido CSM tracking tables
+DB_CREATE_LIDO_CSM_NODE_OPERATORS = """
+CREATE TABLE IF NOT EXISTS lido_csm_node_operators (
```

Comment:
Just making sure that for both those tables each and every column entry is used and we don't keep data for no reason?